theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -0,0 +1,126 @@
+let fundingAgenciesApp = new Vue({
+    el: '#funding-agency-app',
+    delimiters: ['${', '}'],
+    components: {
+        VueBootstrapTypeahead
+    },
+    data: {
+        agencyName: '',
+        fundingAgencies: RES_FUNDING_AGENCIES, // funding agencies from the backend/Django
+        resourceId: SHORT_ID,
+        fundingAgencyNames: [],
+        selectedAgency: null, // keep track of the selected agency from the dropdown
+        crossrefFunders: [], // array of funders to be filled from crossref api
+        CROSSREF_API_URL: 'https://api.crossref.org/funders?query=:query',
+        MIN_SEARCH_LEN: 3, // min # of chars before running a query
+        DEBOUNCE_API_MS: 500, // debounce api requests
+        timeout: null, // used for debouncing
+        showIsDuplicate: false, // error for duplicate funders
+        error: '',
+        isPending: false, // is api fetch pending
+        mode: null, // mode for the modals -- Add or Edit
+        currentlyEditing: {}, // store the funder that we are editing
+        deleteUrl: "", // Django endpoint to call for deleting a funder
+        currentlyDeleting: {} // store the funder that we are deleting
+    },
+    mounted(){
+        this.fundingAgencyNames = this.fundingAgencies.map(a => a.agency_name);
+    },
+    methods: {
+        getCrossrefFunders: async function(query) {
+            if (query === "" || this.showIsDuplicate){
+                return
+            }
+            this.isPending = true
+            // https://api.crossref.org/swagger-ui/index.html#/Funders/get_funders
+            query = query.replace(" ", "+")
+            query = `${query}&mailto=help@cuahsi.org`
```

Comment:
we include the `mailto` so that crossref handles our requests with a privileged set of servers.